app/components/lazy_tag_facet_component.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
 class LazyTagFacetComponent < ViewComponent::Base
-  def initialize(**); end
```

Comment:
I was always conflicted about this semi-colon usage...

Reply:
same.

Reply:
Agree!